### ISSUE TITLE
How to manage the Scrollview Offset when tokenview is added in a scrollview?

### ISSUE THREAD

**Issue by samarth220194** [khawars/KSTokenView#101]
When I add the TokenView in a scroll view, then after adding a certain amount of tokens, I am not able to see the search table view to add more token, because the scroll view content offset is not set accordingly, How to manage this problem?

Here is the screenshot : 
![simulator screen shot - iphone 6s - 2018-03-10 at 17 08 52](https://user-images.githubusercontent.com/17960987/37241833-c39697b6-2485-11e8-9710-996699917f16.png)

**Comment by RazvanCosmeanu** [comment]
You should fork the library and make your changes, the maintainer is not open to new functionality -- see the 5 pending pull requests. It is not to say I'm not grateful for the library, I think it's awesome, I just found it better to download it and make it part of the project if you want custom functionality.

**Comment by samarth220194** [comment]
Yes, I am working on the library and making my own changes, and now I am successful with my work.
